packages/@react-native-windows/cli/src/config/projectConfig.ts

```diff
@@ -115,7 +116,8 @@ export function projectConfigWindows(
       result.solutionFile =
         'Error: Solution file is null in react-native.config.';
     } else {
-      result.solutionFile = userConfig.solutionFile;
+      result.solutionFile = path.normalize(userConfig.solutionFile!);
+      validSolution = true;
```

Comment:
is this right? we don't set validSolution back to false if one of the conditions below fails

Reply:
The indentation from lint is confusing, but that line to set to true is in the `if (usingManualOverride)` block, and the later conditions are part of the `else` block.

Reply:
I believe that path.normalize() will convert forward slashes to back slashes on Windows, but will not do the other way around on Unix systems. I had to add my own logic to do this in a separate project.

Reply:
Since we're blocking the command from running on non-windows anyway, this normalization is mostly just to clean up unnecessary things like `project\..\project\`

Reply:
I mean the new `validSolution = true` in 120, versus the error conditions in 124-149

Reply:
124-149 are checking for project errors and set `validProject`, which is independent from the solution checks that set `validSolution`